release-blog/2024-beta/2024-06-13-beta.0.mdx

```diff
@@ -16,6 +16,21 @@ Overture's `2024-06-13-beta.0` release is available in GeoParquet and stored on
 
 Depending on feedback, we anticipate moving to a general availability (GA) release soon.
 
+### New data: GERS changelog
```

Comment:
What's the timeline for dropping the name "GERS" and replacing "GERS ID" with "Overture ID"?

Reply:
unknown. Right now I'm hedging by calling the `id` Overture ID and the system GERS.